.azure-pipelines/linux/basic-tests.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Azure Pipelines configuration for running tests and other checks on Linux/macOS.
+#
+
+steps:
+  # master branch is needed for Prettier step
+  - script: git fetch origin master:master
+    displayName: 'Fetch master branch'
+
+  - script: node ./scripts/prettier/index.js
+    displayName: 'Prettier'
+
+  - script: node ./scripts/tasks/flow-ci
+    displayName: 'Flow CI'
+
+  - script: |
```

Comment:
I'd switch this to `bash` since it uses bash-specific keywords (which would fail if this step template is ever used on Windows). Of course, someone could always fix this later, but I've generally tried to use "bash" instead of just "script" when the script absolutely requires bash.